docs/articles_en/openvino_workflow/model_optimization_guide/weight_compression.md

```diff
@@ -28,6 +30,32 @@ The code snippet below shows how to compress the weights of the model represente
 
 Now, the model is ready for compilation and inference. It can be also saved into a compressed format, resulting in a smaller binary file.
 
+- **INT4 weight quantization** - this method stands for an INT4-INT8 mixed-precision weight quantization where INT4 is considered as a primary precision and INT8 is a backup one. It usually results in a smaller model size and a lower inference latency but accuracy degradation is also higher. The method has several parameters that can provide different performance-accuracy trade-offs after optimization:
```

Comment:
suggest to replace to "accuracy degradation could be higher, depending on the model"

Reply:
Done